sympy/physics/control/lti.py

```diff
@@ -249,6 +252,147 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dcgain(self):
+        """
+        Returns the zero-frequency (or DC) gain. It is the value
```

Comment:
DC gain is low-frequency gain. 0-frequency response is not defined for unstable systems (blows up). And frequency implies the Fourier transform so it gets all mixed up. I would suggest something like the following 

```python
"""
Computes the low-frequency gain of the frequency response in the limit as frequency
approaches to zero. For systems with pure integrator terms, DC gain is infinite.
"""